Jenkinsfile

```diff
@@ -16,6 +16,8 @@ ansiColor('gnome-terminal') {
         junit(allowEmptyResults: true, testResults: 'target/test-reports/*.xml')
         junit allowEmptyResults: true, testResults: 'target/test-reports/*integration/*.xml'
         archive includes: 'sandboxes.tar.gz'
+        archive includes: 'ci.tar.gz'
+        archive includes: 'ci.log'  // Only in case the build was  aborted and the logs weren't zipped
```

Comment:
Hm, this is done each time now. I'd just do it if the `ci.tar.gz` does not exist. It seems we do not have the `timeout` directive anymore.

Reply:
Done. The `ci.log` is removed if the run was successful.